libqtile/lazy.py

```diff
@@ -93,14 +96,22 @@ def kwargs(self) -> dict:
 
     def when(
         self,
+        check: bool | None = None,
+        *,
```

Comment:
Nice.

Is this a potentially config breaking change? I guess only for the case where people pass an argument to `focused` but without the key word?

Reply:
That's a good point.

I wouldn't expect it to be a config breakage as I would have thought that users would always be using keyword args here. However, that's just an expectation!

A migrate script would be no good here either. 

Perhaps, the best idea is to use a keyword for the Boolean test too. Maybe `condition`?